about.html

```diff
@@ -13,52 +13,75 @@
 
 <body>
     <div class="container about-page">
-     
-    About
+      <header>
+        <img src="img/lambda-black.png">
+        <nav>
```

Comment:
In addition to using list elements you can also remove the list elements and just use nav a